Resources/Textures/_Moffstation/PersonalItems/Wearables/Pixel8/Marvin_Dugmore/marvinheadphones.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Original sprite taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2da, icon and equiped-ears drawn by Pixel8 (SS14) / !FOO#7118/pixellavalamp8 (discord). Please for the love of god tell me if i did this wrong, I don't want to get sued",
```

Comment:
If the original headphone sprites say they're from that commit and that they're the same CC license as this, you're good.
Remove the superfluous bit, please :)